### ISSUE TITLE
Folder only have one sub folder and can't rename

### ISSUE THREAD

**Issue by minspy** [microsoft/vscode#190867]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.81.1 (Universal)
- OS Version: 13.3.1 (a) (22E772610a) Apple M1

Steps to Reproduce:

1. Rename a folder that is the only child of parent
2. can't rename the folder
3. if do 1 , then all file can't open. unless restart vscode.

![2023-08-21 18 16 20](https://github.com/microsoft/vscode/assets/20208875/b1617e5e-77a3-4cb3-bc8d-fb5d78ea5832)

**Comment by gjsjohnmurray** [comment]
/duplicate of https://github.com/microsoft/vscode/issues/188571 which is already fixed in 1.82 Insiders

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for [similar existing issues](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+). See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
